main.go

```diff
@@ -1,6 +1,6 @@
 package main
 
-import "cli/cmd"
```

Comment:
what's the significance of this change?

Reply:
oh I see you standardized the module name

Reply:
In the package imports local to runpodctl, should the imports be the fully qualified `github.com/...` names or just local paths? I'm unfamiliar with go best practices or any reason to do one or the other

Reply:
they should always be the fully qualified name. ideally, you should be able to know where to find all the dependencies for a file by just reading the import statement and putting `https://` in front.

Reply:
Alright, that makes sense